egs/wsj/s5/steps/nnet3/lstm/train.sh

```diff
@@ -147,7 +149,7 @@ if [ $# != 4 ]; then
   
   exit 1;
 fi
-
+set -x
```

Comment:
Why are you doing this set -x?  My understanding is that it exports bash variables without using the 'export' command.  I don't think silently exporting variables is a good idea.